bugwarrior/services/redmine.py

```diff
@@ -1,23 +1,30 @@
 import six
 import requests
+import re
 
 from bugwarrior.config import die
 from bugwarrior.services import Issue, IssueService, ServiceClient
+from taskw import TaskWarriorShellout
 
 import logging
 log = logging.getLogger(__name__)
 
 
 class RedMineClient(ServiceClient):
-    def __init__(self, url, key, auth):
+    def __init__(self, url, key, auth, issue_limit):
         self.url = url
         self.key = key
         self.auth = auth
+        self.issue_limit = issue_limit
 
-    def find_issues(self, user_id=None):
-        args = {"limit": 100}
-        if user_id is not None:
-            args["assigned_to_id"] = user_id
+    def find_issues(self, issue_limit=100):
+        args = {}
+        # TODO: if issue_limit is greater than 100, implement pagination to return all issues.
+        # Leave the implementation of this to the unlucky soul with >100 issues assigned to them.
+        if issue_limit is not None:
+            args["limit"] = issue_limit
+
+        args["assigned_to_id"] = 'me'
```

Comment:
Yes, but, should it be conditional based on whether the user has selected `only_if_assigned = True` in their config?